### DISCUSSION TITLE
Contributing to react-spring.io

### DISCUSSION THREAD

**Discussion by oanaOM** [pmndrs/react-spring#1476] [Support] [answered]
Hey all,

Wanted to make a small contribution to one of the pages of https://react-spring.io/ repo and when I try to push the changes to my new branch created, I get this error 403 forbidden permission:
![image](https://user-images.githubusercontent.com/15228388/116468009-41446a80-a868-11eb-93e0-5b36542cc37a.png)

Was wondering if this is because it's not open sourced?

Thanks!

**Comment by joshuaellis** [comment, marked as answer]
Hi @oanaOM,

You don't have specific access to the `react-spring.io` repo, which isn't uncommon! I don't have access to loads of them, like `react` or `typescript`. But they're still open source because you can do the following – create a fork and push your changes to said fork. Afterwards you can create a PR across forks.
